src/i18n/cs_CZ/vehicles.ts

```diff
@@ -214,104 +292,175 @@ export default {
         staff: {
             min: 2,
             max: 3,
+            training: {
+                'Policie': {
+                    fbi_bomb_tech: {
+                        all: false,
+                    },
+                },
+            }
         },
         icon: 'car-side',
-        possibleBuildings: [],
+        possibleBuildings: [21],
     },
     20: {
         caption: 'Přívěs se člunem',
         color: '#990000',
         credits: 6000,
         coins: 12,
         staff: { min: 0, max: 0 },
-        special: 'Požadováno po vybudování 11 požárních stanic',
-        icon: 'car-side',
-        possibleBuildings: [],
+        special: 'Je potřeba tažné vozidlo (CAS 20, Potápěčský automobil, DA) s minimální posádkou 3 členů vyškolenou v oboru (Záchrana z vody).',
+        icon: 'trailer',
+        possibleBuildings: [0, 18],
+        isTrailer: true,
+        tractiveVehicles: [0, 22, 30],
     },
     21: {
         caption: 'Přívěs se člunem VZS ČČK',
         color: '#990000',
         credits: 6000,
         coins: 12,
         staff: { min: 0, max: 0 },
-        special:
-            'Je potřeba tažné vozidlo (SUV VZS ČČK, Dodávka VZS ČČK). Vyžaduje osoby(4) se speciálním vzděláním v oboru odtahové vozidlo',
-        icon: 'car-side',
-        possibleBuildings: [],
+        special: 'Je potřeba tažné vozidlo (SUV VZS ČČK, Dodávka VZS ČČK). Vyžaduje osoby(4) se speciálním vzděláním v oboru (Vodní záchranná služba)',
+        icon: 'trailer',
+        possibleBuildings: [15],
+        isTrailer: true,
+        tractiveVehicles: [23, 24],
     },
     22: {
         caption: 'Potápěčský automobil',
         color: '#990000',
         credits: 10_000,
         coins: 25,
-        staff: { min: 2, max: 4 },
-        special: 'Požadováno po vybudování 11 požárních stanic',
+        staff: {
+            min: 2,
+            max: 4,
+            training: {
+                'Hasiči': {
+                    gw_taucher: {
+                        all: false,
+                    },
+                },
+            }
+        },
         icon: 'car-side',
-        possibleBuildings: [],
+        possibleBuildings: [0, 18],
     },
     23: {
         caption: 'SUV VZS ČČK',
         color: '#990000',
         credits: 10_000,
         coins: 25,
-        staff: { min: 1, max: 4 },
+        staff: {
+            min: 1, 
+            max: 4 
```

Comment:
missing `,` here :upside_down_face:

Reply:
Thank you for highlighting the errors. 😊